Jenkinsfile

```diff
@@ -0,0 +1,250 @@
+                    tfausBkbWRzM7yfsdKhxsCRN5o0NI9bHPpcDp4mf9In4+L/e4+vFPutgjY46Fgvh
+                    Tlx8yRs9tTNXAPRvZ6OoILxK3htQb2WQ3GG6dGE6Of2D/wKUcH2IVf0tpB49vIMh
+                    8jIGiF9Q8XoD2shjUxbu4oQZ4+VjT3VX5RYNHbgFiUwLcviCfyyFJefxGkGB+0OF
+                    a3BBRaovzKhVDrJmglWvn1GVTShHTQARAQAB/gcDAs+H1q9eFzirYIpK191FegPa
+                    ztcbJBMNMNTw1XpSt4E0z0fmYi/Mn66FMXQvM4j7mE9TnGq6f5jXxZzAF0FRLHdZ
+                    4N3I04dkdgsaoTTY7PsIYMd0NwDtrr33P6HFefrz3kEf+qor8hZkI73Pts5Md/7L
+                    cTZ4q0gK0BFo9zXddBGq0J/c3ojXS8pH37xetZPmJgr9EfBjvLB5gcxCU6+Zsdnj
+                    rGqBCRAwzZf/+4p4OQUtJfIL9i0i+Ek6gjI2ZMaZ70V/wMwDL+Tnl0Ftk+CAvKbl
+                    qYOwlsOOLvp/9FibEJAUXvgjVQQPzS/lzvXfoxkR9LgGzYWk1pp8HWzwhwElhjm1
+                    +0f2FNhbckuy3VXXg+CqIr+zL43FnRrsf1+YAYmUUrW5sZpe4Z/A9PfRDMgCVXlv
+                    XCd/iv679XIK3l3TljO98/nlEYj4HB6pGicjmkFuF5tqwP5sY/D9y5i72LxisPHp
+                    y4seVpoM6590hHld4bt8UnCU8ZJp3qxuwJG5RHNznp9caNwDNxhVqe74yLsffHUO
+                    /qzrb9yAr5EYYuH8ZgZfwMwoI7NCUg+dVobLkF3bQbwrbO4IgUv5N9x12UwO0XeQ
+                    AiVtPeiQD4cLBALHSeVkRrMm72SyElp8a0B4Do8Lb7xWKSRQ4+/v08S1kI2mBFWj
+                    0+Y3ydfEDGS75yzdzsdzkQEgOsk3GmYoLRJtzCCsDjHCOg66EN62g/TN7V2rsqpq
+                    6J+GM+JAgFJNuCf9aqXJ4k/dWJaCD2n3BkL9CTB1SSeFifQwZa0uwGoZ2+i+Og06
+                    Ie6Ge1LPlyhKGugRC9gH+ufet3LeeePN070yAL9IpmzC5SR20YjH95Sf6zt6PD8W
+                    y6ZmFRcGjtZcTDitY0P6LXUCrbm7cYYp4x3+sWJDZ0XgpIHWtDmPp9FUQAbJurmY
+                    aVvNc4kBHwQYAQIACQUCV4/RPwIbDAAKCRCeSGoTZ8VKXiTwB/49TRwFs1qQNSKk
+                    DB3213PpVMEcyRwXlPQrZfHlPIDbyM0x028tsNLbBD2xYsYiBqd8TljJz+0Nt5If
+                    HxxDXnfyxyuLmmKVIt1S/28d0HB3iFr0c3lRBLg1hjZw1Rv5J/hUFKS6wgi1C+Cn
+                    c9+MUMLMto9a0+G94FJTHnZCuLDjroI6t0/PoNs7xdd8DeDQdRrgAXUJZnin1MGK
+                    rLZazkL9FfGoftJspDdrcVFYXEQo5PWs/xffjeNzKYQmdCyNs45SEFwnJOUXjP9m
+                    BUeoJPFwTAkJ4nOo5epXPgNrfRcH/4h8KPkJPoOEOw3vmu4yMUtnjA3gi01YrELA
+                    gCDiewHZ
+                    =4nEF
+                    -----END PGP PRIVATE KEY BLOCK-----
+                    " > secret.asc
+                        gpg --import secret.asc
+                        echo "done importing"
+                    fi
+
+                    gpg --list-secret-keys
+
+                    echo "<settings>
+                    <servers>
+                        <server>
+                        <id>ossrh</id>
+                        <username>tim.smith</username>
+                        <password>5yKee3%5I@^XDRSxMd</password>
+                        </server>
+                    </servers>
+                    </settings>" > settings.xml
+
+                '''
+            }
+        }
+
+        stage("build") {
+            steps {
+                sh '''
+                    echo "***RONAN***"
```

Comment:
Looks like I left these 2 lines in for debugging. They can be removed :)

Reply:
Removed